solana/write-account/src/lib.rs

```diff
@@ -11,28 +16,47 @@ solana_program::entrypoint!(process_instruction);
 /// The first byte of the `instruction` determines operation to perform.  Format
 /// of the instruction and required accounts depend on that.
 ///
+/// # Create PDA Account
+///
+/// Instruction with discriminant zero is Create and its format is as follows:
+/// ```ignore,text
+/// +-----+---------------+------------+
+/// | 0u8 | data_len: u32 | data: [u8] |
```

Comment:
Should this read seed?

```suggestion
/// | 0u8 | account_size: u32 | seed: [u8] |
```

Reply:
Sorry my bad, it reads seed only.